posthog/migrations/0299_add_constraints_to_person_override.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 3.2.16 on 2023-02-02 14:32
+
+import django.db.models.expressions
+from django.db import migrations, models
+
+from posthog.models.person import PersonOverride
+
+
+class Migration(migrations.Migration):
```

Comment:
We will need to ensure that this migration is also followed up with by the new logical replica. cc @guidoiaquinti @hazzadous

Reply:
Suggest we wait for the replica to be up to date before making any schema changes.

Reply:
Will wait, thank you.